src/hmf/cosmology/growth_factor.py

```diff
@@ -163,6 +163,7 @@ def growth_factor_fn(self, zmin=0.0, inverse=False):
     def growth_rate(self, z):
         """
         Growth rate, dln(d)/dln(a) from Hamilton 2000 eq. 4
+        Note that the growth function g in Hamilton 2000 is defined as g=D*(1+z).
```

Comment:
While we're at it, can we update the docs to point to the arxiv or DOI of the paper? (https://arxiv.org/pdf/astro-ph/0006089.pdf)

Reply:
Thanks for the feedback! You meant implement:

- A similar fix to the growth rate method in `GenMF`? And
- A python built-in `UserWarning` for each method?

I can do so, but maybe we should just merge this with the fix to the issue I mentioned in #198?

Reply:
I think that's a good idea -- it might be best to figure out what's going on in #198 and add the fix to this PR. I think the fix for `GenMF` should not be difficult, and neither should the warning (perhaps you can add it to the base class). I'm still a little hesitant on the growth rate differences between CAMB and the numerical integral. I'd like to understand it better, but I have no time in the next couple of weeks.